README.md

```diff
@@ -18,14 +18,14 @@ Since Camel-K already does a pretty good job building Camel applications in any
 
 ## How does it work?
 
-Without going into many details, Camel-K has basically two phases of their build, which I call "Project Assemble" and "Image Build". 
+Without going into many details, Camel-K has basically two phases of their build, which I call "Project Assemble" and "Image Build".
 In this first phase, Camel-K reads the Route configuration, assemble the Maven project and run the project build.
 Then it takes the Java application and build into an image.
 
 Camel-K has this concept of "Environment Platform" that based on the type of cluster which is running, can pick the right build feature.
 For example, source to image on OpenShift clusters or Kaniko on Kubernetes.
 
-Kogito might use this first phase to assemble a specific application based on the sources pushed to the cluster. 
+Kogito might use this first phase to assemble a specific application based on the sources pushed to the cluster.
```

Comment:
what first phase?

Reply:
https://github.com/kiegroup/container-builder/pull/10/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R23

Reply:
this link points to nothing.
the first phase I assume that is project assemble, but reading this paragraph it turns a little bit confusing.

Reply:
"Camel-K has basically two phases of their build, which are "Project Assemble" and "Image Build"." https://github.com/kiegroup/container-builder/pull/10/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R23